spec.bs

```diff
@@ -5345,8 +5404,9 @@ result of [=evaluating a bidding script=], or an [=additional bid=] provided by
   : <dfn>ad descriptor</dfn>
   :: An [=ad descriptor=]. Render URL and size of the bid's ad.
   : <dfn>ad component descriptors</dfn>
-  :: Null or a [=list=] of [=ad descriptors=]. Ad components associated with bid, if any. May have at
-    most 40 [=list/items=]. Must be null if the interest group making this bid has a null
+  :: Null or a [=list=] of [=ad descriptors=]. Ad components associated with bid, if any.
+    Will be limited to at most 40 [=list/items=] by the time the bid participates in the
+    auction. Must be null if the interest group making this bid has a null

```

Comment:
mention it's also limited to at most {{GenerateBidOutput/targetNumAdComponents}}"] if generate bid returns one?

Reply:
Done --- it's actually exactly that many if the target is set.